inc/ses_to_cloudwatch/namespace.php

```diff
@@ -43,3 +53,21 @@ function on_error_sending_message( Exception $error, $message ) {
 		'message' => $message,
 	] ) );
 }
+
+/**
+ * Truncate string to given maximum size.
+ *
+ * @param string $string      String message to truncate.
+ * @param int    $max_size    Maximum size in bytes, default 5KB.
+ * @param string $replacement String replacement.
```

Comment:
We don't align docblock params any more, just prevents unnecessary code changes when adding / changing a parameter in future.
```suggestion
 * @param string $string String to truncate.
 * @param int $max_size Maximum size in bytes, default 5KB.
 * @param string $replacement String replacement.
```